bool turn_on
bool turn_off
---
bool is_on